### ISSUE TITLE
/www/GlucoseFreedom.js Not found

### ISSUE THREAD

**Issue by BilelKorbosli** [hrishioa/Juventas-Ionic#1]
the plugin is missing this file
'Juventas-Ionic/plugins/cordova.plugin.glucosefreedom/www/GlucoseFreedom.js'

**Comment by hrishioa** [comment]
Did adding the plugin step in the installation finish correctly? What does `ionic cordova plugin add GlucoseFreedom` do?

It looks like an issue with the linking of the plugin, which sometimes happens when you're on a different version on Ionic.

**Comment by BilelKorbosli** [comment]
yes I did all the steps right but the plugin in the github repository missing www/GlucoseFreedom.js
[https://github.com/hrishioa/Juventas-Ionic/tree/master/GlucoseFreedom](https://github.com/hrishioa/Juventas-Ionic/tree/master/GlucoseFreedom)

**Comment by hrishioa** [comment]
Can you post the full command and error message? Are you getting this on adding the plugin or on compilation?
